spec/models/articles_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe 'validations' do
+    before { subject.valid? }
+    describe 'title' do
+      context 'when blank' do
+        subject { build(:article, title: nil) }
+        it { expect(subject.errors[:title]).to include("can't be blank") }
+      end
+
+      context 'when less than 5 characters' do
+        subject { build(:article, title: 'name') }
+        it { expect(subject.errors[:title]).to eq(['is too short (minimum is 5 characters)']) }
```

Comment:
Here we have `.to eq` in previous test  `.to include`. Looks like we have a bug in model, cause it returns 2 failed validations at the same time

Reply:
If add `allow_blank: true ` to the :title validation, The test with `eq`, instead of `include`, passes. So should this bug be fixed in that way?